jquery/Q_4.html

```diff
@@ -0,0 +1,57 @@
+<html>
+<head>
+  <title>Exercise 5 - rampup</title>
+  <style type="text/css">
+    #countryParent, #countryChild {
+      width: 200px;
+      height: 100px;
+    }
+
+  </style>
+  <script
+  src="https://code.jquery.com/jquery-3.4.1.min.js"
+  integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
+  crossorigin="anonymous"></script>
+  <script>
+    function addOptions(from, to)
+    {
+      while($("#" + from).prop('selectedIndex') != -1)
+      {
+        var val = $("#" + from).val();
+        var optionNumber = $("#" + from).prop('selectedIndex')
+        var Text = $( "#" + from + " option:selected" ).text();
+        $('#' + to).append($('<option>', {
+            value: val,
+            text: Text
+        }));
+        document.getElementById(from).remove(optionNumber);
```

Comment:
don't use `document.getElementbyId` or `document.getElementsByClassName` when you are using jquery